MagicBytesValidator.CLI/MagicBytesValidator.CLI.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
You can also write a Directory.Build.Props that contains all the information for all projects within your solution that share it.
You should do that.